package-lock.json

```diff
@@ -5316,9 +5297,9 @@
           "dev": true
         },
         "fsevents": {
-          "version": "2.0.7",
-          "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-2.0.7.tgz",
-          "integrity": "sha512-a7YT0SV3RB+DjYcppwVDLtn13UQnmg0SWZS7ezZD0UjnLwXmy8Zm21GMVGLaFGimIqcvyMQaOJBrop8MyOp1kQ==",
+          "version": "2.1.0",
```

Comment:
Don't think we need this change?

Reply:
probably not - this pr won't go anywhere near one of the base branches , so I'm not too bothered about this one :)